app/services/signature-service.js

```diff
@@ -43,6 +43,8 @@ export default class SignatureService extends Service {
       // set creator
       const creator = await this.currentSession.user;
       signFlow.creator = creator;
+      const status = await this.store.findRecordByUri('concept', PREPARED);
+      signFlow.status = status;
```

Comment:
If we do this, we need to set the status back to MARKED after `uploadPiecesToSigninghub` fails